scripts/unsubscribe_inactive_people.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+include Rails.application.routes.url_helpers
```

Comment:
Can you try moving this into the `unsubscribe` method? See if that works

Reply:
if u @donjar see one commit before this, which is "fix undefined path", i put it inside ```unsubscribe``` method but the error is undefined method include. so i put it outside and found this error, i guess